Misc/NEWS.d/next/Core and Builtins/2022-05-20-13-32-24.gh-issue-93012.e9B-pv.rst

```diff
@@ -0,0 +1,13 @@
+Added the new function :c:func:`PyType_FromMetaModuleAndSpec`, which
+generalizes the existing :c:func:`PyType_FromModuleAndSpec` using an additional
+metaclass argument. This is useful for language binding tools, where it enables
+use cases including
+
+- storing additional binding-related data structures in the type object.
+
+- intercepting type-related operations like subclassing or static attribute
```

Comment:
reST formatting fix:

```suggestion
* storing additional binding-related data structures in the type object.

* intercepting type-related operations like subclassing or static attribute
```

Reply:
The docs check should pass now. Hope you didn't mind me fixing those, @wjakob.

Reply:
Not at all, thank you very much!